src/js/components/Grommet/doc.js

```diff
@@ -31,3 +31,17 @@ export const doc = Grommet => {
 
   return DocumentedGrommet;
 };
+
+export const themeDoc = {
+  'grommet.extend': {
+    description: 'Any additional style for Grommet.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },  
+  'global.font.face': {
+    description: 'Custom font face declaration',
```

Comment:
@alansouzati following our conversation, making sure this is what you had in mind.